Casbin/Abstractions/IEnforcer.cs

```diff
@@ -76,11 +88,7 @@ public IEnumerable<bool> BatchEnforce<TRequest>(EnforceContext context, IEnumera
 
         #region Options
 
-        public bool Enabled { get; set; }
-        public bool EnabledCache { get; set; }
-        public bool AutoBuildRoleLinks { get; set; }
-        public bool AutoNotifyWatcher { get; set; }
-        public bool AutoCleanEnforceCache { get; set; }
```

Comment:
I think we need not to remove those property, lt may be too many beaking changes.